src/common/Config.ts

```
@@ -146,7 +146,13 @@ export class CommonConfig {
       return Promise.resolve();
     }
 
-    const addressAggregator = new AddressAggregator([new addressAdapters.fs.AddressList(path)], logger);
+    const addressAggregator = new AddressAggregator(
+      [
+        new addressAdapters.fs.AddressList({ path }),
+        new addressAdapters.risklabs.AddressList({ throwOnError: false }),
```

tldr try to pull down the latest address list (permitting up to 1 retry by default), but suppress errors if it can't be retrieved. In this case, the build-time `addresses.json` should still include a reasonably-recent list of addresses that can be used as a fallback.